src/UI/Implementation/Component/MainControls/Slate/Slate.php

```diff
@@ -66,7 +72,8 @@ public function __construct(
     protected function initSignals()
     {
         $this->toggle_signal = $this->signal_generator->create();
-        $this->show_signal = $this->signal_generator->create();
+        $this->engage_signal = $this->signal_generator->create();
+        $this->replace_signal = $this->signal_generator->create("ILIAS\\UI\\Implementation\\Component\\ReplaceSignal");
```

Comment:
```suggestion
        $this->replace_signal = $this->signal_generator->create(ILIAS\\UI\\Implementation\\Component\\ReplaceSignal::class);
```
This is not just any string, but a class name.